opennebula/resource_opennebula_virtual_network.go

```diff
@@ -384,29 +407,39 @@ func resourceOpennebulaVirtualNetworkCreate(d *schema.ResourceData, meta interfa
 	controller := meta.(*goca.Controller)
 	var vnc *goca.VirtualNetworkController
 
+	reservation_vnet := d.Get("reservation_vnet").(int)
+
 	// VNET reservation
-	if rvnet, ok := d.GetOk("reservation_vnet"); ok {
-		reservation_vnet := rvnet.(int)
-		reservation_name := d.Get("name").(string)
-		reservation_size := d.Get("reservation_size").(int)
-
-		if reservation_vnet < 0 {
-			return fmt.Errorf("Reservation VNET ID must be greater than 0!")
-		} else if reservation_size <= 0 {
-			return fmt.Errorf("Reservation size must be strictly greater than 0!")
+	if reservation_vnet > -1 {
```

Comment:
Could you rename all variable you modify with Go naming style ?

`reservation_vnet` -> `reservationVnet`
`reservation_first_ip` -> `reservationFirstIP`
`reservation_ar_id` -> `reservationARID`
...